qlib/utils/__init__.py

```diff
@@ -742,3 +744,55 @@ def load_dataset(path_or_obj):
     elif extension == ".csv":
         return pd.read_csv(path_or_obj, parse_dates=True, index_col=[0, 1])
     raise ValueError(f"unsupported file type `{extension}`")
+
+
+def set_config(config_c, default_conf="client", **kwargs):
```

Comment:
Will it be better if this is a static methods in class Config?

Reply:
The name of the function could be greatly simplified if it is a static methods in `class Config`

Reply:
You can even change  `config_c` to `self`